config/logback.xml

```diff
@@ -11,14 +11,35 @@ You may obtain a copy of the License at
 <configuration scan="true" scanPeriod="30 seconds">
     <logger name="org.apache.zookeeper" level="WARN"/>
 
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <file>standalone.log</file>
+        <encoder>
+            <charset>UTF-8</charset>
+            <Pattern>%d %-4relative [%thread] %-5level %logger{35} - %msg%n</Pattern>
+        </encoder>
+    </appender>
+
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
             <charset>UTF-8</charset>
             <Pattern>%d %-4relative [%thread] %-5level %logger{35} - %msg%n</Pattern>
         </encoder>
     </appender>
 
-    <root level="DEBUG">
+    <logger name="io.pravega.local" level="INFO">
```

Comment:
The idea with this change was to bring forward two things: readiness of various services (zookeeper, controller, segmentstore, etc) and highlight the configuration options of those services. It was separated from normal logging (which now is directed towards `standalone.log`) to ensure visibility of these things.

Reply:
Wouldn't be better to define all the necessary changes to logging for standalone in `logback-standalone.xml` file and use it whenever Standalone starts (like in a gradle file)?  

otherwise these changes will be taking an effect in production as well and I am not sure how they fit K8s environments

Reply:
Good point. There are a number of different logback files spread throughout our codebase. We have to be sure that the correct one is picked up by Logback at runtime. We should also make sure this change doesn't get picked up when we publish the production jars for releases.

Reply:
I thought that production environments use the `logback.xml` from `dist/conf/`? Although it does seem good to have a more explicitly named `logback.xml` file.